tabs/src/components/app/views/DocumentationLinks.jsx

```diff
@@ -1,28 +1,39 @@
 import React from "react";
 import { UNLOCALIZED_GRAPH_EXPLORER_URL, UNLOCALIZED_GRAPH_EXPLORER_DOCS_URL, UNLOCALIZED_OFFICIAL_RSC_URL, localizeUrl } from "../../TabConstants";
-import { Button } from '@fluentui/react-northstar';
 import { useTranslation } from 'react-i18next';
+import { useTeamsFx } from "../../utils/useTeamsFx";
+import { MediumFontLink } from "./MediumFontLink";
 
 export const DocumentationLinks = () => {
     const { t, i18n } = useTranslation();
+    const { themeString } = useTeamsFx();
+
+    let linkColor = "#6264A7";
+    switch(themeString) {
+        case "default":
+            linkColor = "#6264A7";
+            break;
+        case "dark":
+            linkColor = "#A6A7DC";
+            break;
+        case "contrast":
+            linkColor = "#ffff01";
+            break;
+        default:
+            linkColor = "#6264A7";
+    }
```

Comment:
you could move this inside `MediumFontLink` then u dont need to pass link color

Reply:
Tried making this change, but it made the color of the hyperlinks "laggy" when the `Documentation Links` section was expanded, i.e. you could see the default hyperlink color for a split second before it changed to the correct `Dark` or `High contrast` color.

Reply:
makes sense!